pom.xml

```diff
@@ -69,7 +69,7 @@
         <workflow-support-plugin.version>2.17</workflow-support-plugin.version>
         <scm-api-plugin.version>2.0.8</scm-api-plugin.version>
         <groovy-cps.version>1.24</groovy-cps.version>
-        <jenkins-test-harness.version>2.33</jenkins-test-harness.version>
+        <jenkins-test-harness.version>2.37</jenkins-test-harness.version>
```

Comment:
file a request to update the parent POM

Reply:
@jglick  https://github.com/jenkinsci/plugin-pom/pull/101 - Done